packages/asset-swapper/src/types.ts

```diff
@@ -192,6 +193,10 @@ export interface SwapQuoteOrdersBreakdown {
  */
 export interface SwapQuoteRequestOpts extends CalculateSwapQuoteOpts {
     gasPrice?: BigNumber;
+    takerAddress?: string;
```

Comment:
Consider pushing these down into `rfqt` since they're all grouped together and only make sense in that context.

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2541/commits/70add44fc1b5418fab24274beb9c7692bc4c8859